dynamic.js

```diff
@@ -285,6 +285,10 @@ module.exports = function (fastify, opts, next) {
           responsesContainer[key] = {
             schema: resolved
           }
+          if (resolved.description) {
+            responsesContainer[key].description = resolved.description
+            delete resolved.description
```

Comment:
I fear that this will change the schema of the user, we could just avoid to delete this field and the swagger output should just work as now

Reply:
@Eomm so... can I remove the line 290?

Reply:
Could you test it?

If it is true, that the json object of the user is changed, I would remove the line since it won't affect the swagger API interface